src/picongpu/include/traits/attribute/GetChargeState.hpp

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2014 Rene Widera
```

Comment:
NEVER! ... well, actually Rene didn't even create this file. :stuck_out_tongue: 
I lied outright there.